client/components/public/Gear.jsx

```diff
@@ -229,7 +227,7 @@ Gear = class Gear extends Component {
           <PuzzlePageTitle title="Gear" />
           Gear store closes { gearSaleEnd }.
           <br />
-          Shirts will be ordered on Monday, April 11 and assuming no supply chain delays, should be shipped out or ready for pick-up the week of April 18.
+          Shirts will be ordered on Monday, April 17 and assuming no supply chain delays, should be shipped out or ready for pick-up the week of April 24.
```

Comment:
Not for 2023, but maybe in future: should we make more of the gear dates be configuration settings, similar to eventYear?

Reply:
I agree, something like `gearOrderDate` and `gearShipDate`.